src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3491,63 +3508,83 @@ String MusicXMLParserDirection::matchRepeat() const
         return u"daCapoAlFine";
     }
     if (plainWords.contains(daCapoAlCoda)) {
+        setMarkerId(m_codaId, plainWords);
```

Comment:
It would be great to call it outside of matchRepeat() and make the function const again...